internal/controller/certificates/policies/checks_test.go

```diff
@@ -1459,9 +1459,9 @@ func Test_SecretAdditionalOutputFormatsManagedFieldsMismatch(t *testing.T) {
 						ManagedFields: []metav1.ManagedFieldsEntry{
 							{Manager: "not-cert-manager", FieldsV1: &metav1.FieldsV1{
 								Raw: []byte(`
-              {"f:data": {
-							  ".": {},
-							  "f:tls-combined.pem": {}
+							{"f:data": {
+								".": {},
+								"f:tls-combined.pem": {}
```

Comment:
note: these changes are just fixing spaces -> tabs to stop my editor complaining about mixed indentation